### PR TITLE
Update asserted verifier log pattern accepting agent identification

### PR DESCRIPTION
**Pull request by Koncpa**

### CODE REVIEW

File: functional/measured-boot-swtpm-sanity/test.sh [hunk 1]
```diff
@@ -85,7 +85,7 @@ rlJournalStart
         #rlRun "tsseventextend -tpm -if /var/tmp/binary_bios_measurements"
         rlRun -s "keylime_tenant -u $AGENT_ID --verify --tpm_policy '{}' --runtime-policy policy.json -f /etc/hostname -c add --mb_refstate mb_refstate2.txt"
         rlRun "limeWaitForAgentStatus $AGENT_ID 'Tenant Quote Failed'"
-        rlAssertGrep "keylime.tpm - ERROR - For PCR 0 and hash sha256 the boot event log has value '.*' but the agent returned '.*'" $(limeVerifierLogfile) -E
+        rlAssertGrep "keylime.tpm - ERROR - For PCR 0 and hash sha256 the boot event log has value '.*' but the agent .* returned" $(limeVerifierLogfile) -E
```
Comment (kkaarreell) [inline functional/measured-boot-swtpm-sanity/test.sh:88]: With spaces on both sides of `.*` this assert won't match the old message format. You should remove one of them.
Also, please but back the pattern after "returned" if this part is still there. This change is not about different wording but about additional details about the agent added to the log message.

**Review by kkaarreell — APPROVED** [review]
LGTM. Please once tests pass remove the repo redirect to verify it works also with the current upstream and then you can merge it.

**Comment by kkaarreell** [comment]
Test failures on F38 and Rawhide are reported as bug https://bugzilla.redhat.com/show_bug.cgi?id=2171376

**Comment by kkaarreell** [comment]
@Koncpa Please remove the 2nd commit and possibly also reword the 1st one according to the updated PR summary.

**Comment by Koncpa** [comment]
> Test failures on F38 and Rawhide are reported as bug https://bugzilla.redhat.com/show_bug.cgi?id=2171376

Thank you for reporting @kkaarreell .

**Pull request merged**